pkg/kubelet/kubelet_pods.go

```diff
@@ -145,14 +146,14 @@ func (kl *Kubelet) getKubeletMappings() (uint32, uint32, error) {
 	cmd, err := exec.LookPath(execName)
 	if err != nil {
 		if os.IsNotExist(err) {
-			klog.V(2).InfoS("Could not find executable, default mappings will be used for the user namespaces", "executable", execName, "err", err)
+			klog.V(2).InfoS("user namespaces: executable not found, using default mappings", "executable", execName, "err", err)
 			return defaultFirstID, defaultLen, nil
 		}
 		return 0, 0, err
 	}
 	outUids, err := exec.Command(cmd, kubeletUser).Output()
 	if err != nil {
-		return 0, 0, fmt.Errorf("error retrieving additional ids for user %q", kubeletUser)
+		return 0, 0, fmt.Errorf("error retrieving additional uids for user %q", kubeletUser)
```

Comment:
do we need the `user namespaces:` prefix here?

Reply:
Good idea. I've added a wrapping to all the errors to create the user namespace manager, so the errors are clear too :)

Reply:
But for the log lines, the prefix is needed to have all the context :)

Reply:
```suggestion
		return 0, 0, fmt.Errorf("error retrieving additional uids for user %q: %w", kubeletUser, err)
```

Reply:
Oh, great catch. Fixed!